docs/release/notes.md

```diff
@@ -8,6 +8,8 @@ Release Notes
 | 3.3.28            | 12-Sep-2017  | [Detailed Notes](3.3/release-3-3-28)  | TBA                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
```

Comment:
Oops, I forgot to remove 3.3.28 (there's a corresponding entry in the 3.3 table). Could you remove it?

Reply:
OK will do.